app/cisc/cisc.go

```diff
@@ -0,0 +1,312 @@
+package main
+
+import (
+	"os"
+
+	"encoding/hex"
+
+	"path"
+
+	"io/ioutil"
+	"strings"
+
+	"github.com/dedis/cothority/app/lib/config"
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/services/identity"
+	"gopkg.in/codegangsta/cli.v1"
+)
+
+/*
+Cisc is the Cisc Identity SkipChain to store information in a skipchain and
+being able to retrieve it.
+
+This is only one part of the system - the other part being the cothority that
+holds the skipchain and answers to requests from the cisc-binary.
+*/
+
+var configFile string
+var sshDir string
+var sshConfig string
+var clientApp *identity.Identity
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "SSH keystore client"
+	app.Usage = "Connects to a ssh-keystore-server and updates/changes information"
+	app.Commands = []cli.Command{
+		commandID,
+		commandConfig,
+		commandKeyvalue,
+		commandSSH,
+	}
+	app.Flags = []cli.Flag{
+		cli.IntFlag{
+			Name:  "debug, d",
+			Value: 1,
+			Usage: "debug-level: 1 for terse, 5 for maximal",
+		},
+		cli.StringFlag{
+			Name:  "config, c",
+			Value: "~/.cisc",
+			Usage: "The configuration-directory of cisc",
+		},
+		cli.StringFlag{
+			Name:  "config-ssh, cs",
+			Value: "~/.ssh",
+			Usage: "The configuration-directory of the ssh-directory",
+		},
+	}
+	app.Before = func(c *cli.Context) error {
+		configDir := config.TildeToHome(c.String("config"))
+		os.Mkdir(configDir, 0660)
+		log.SetDebugVisible(c.Int("debug"))
+		configFile = configDir + "/config.bin"
+		if err := loadConfig(); err != nil {
+			log.Error("Problems reading config-file. Most probably you\n",
```

Comment:
Suppose the config-file is corrupted and you're about to create a new one with a new cothority. So instead of having to delete the old config-file before creating a new cothority, you can do it in one go.